src/lbaf/Applications/conf.yaml

```diff
@@ -50,7 +50,7 @@ work_model:
   name: AffineCombination
   parameters:
     alpha: 1.
-    beta: 0.
```

Comment:
I think we should not change values in configuration file, this spams the git and makes looking for changes more difficult.

Reply:
Ok restored to default